onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -1495,6 +1495,20 @@ def test_script_function_passes_checker(self, _, func_with_wrangler):
         function_proto = func.to_function_proto()
         onnx.checker.check_function(function_proto)  # type: ignore[attr-defined]
 
+    @parameterized.parameterized.expand(list(OPINFO_FUNCTION_MAPPING_SCRIPTED.items()))
+    @unittest.skipIf(
+        version_utils.onnx_older_than("1.14"),
+        "OpSchema is not writable before ONNX 1.14",
+    )
+    def test_script_function_has_op_schema(self, _, func_with_wrangler):
+        if isinstance(func_with_wrangler, tuple):
+            func = func_with_wrangler[0]
+        else:
+            func = func_with_wrangler
+        op_schema = func.opschema
+        assert op_schema is not None
+        self.assertEqual(op_schema.name, func.name)
```

Comment:
Is there ONNXFunction with wrangler?

Reply:
That was how we setup the tuple list in test: `Tuple[Tuple[<the function>, <wrangler function>]]

Reply:
Yeah, I was thinking if the goal here is to test all ONNXFunction, we could isinstance(func, ONNXFunction) rather than see if it has wrangler or not.

Reply:
So this logic is just taking the onnx function out from the tuple. We are testing all onnx functions already I think?

Reply:
I misread the whole thing. NVM